lib/smartcar/user.rb

```diff
@@ -13,7 +13,7 @@ class User < Base
     # @param token [String] Access token
     #
     # @return [String] User ID
-    def self.user_id(token:, version: DEFAULT_API_VERSION)
+    def self.user_id(token:, version: Smartcar.get_api_version)
```

Comment:
we dont need to document these `version` parameters in readme, but this gives us a little more control to operate with multiple version if needed (looking forwarded to future version migrations slowly like maybe comatibility v3 etc etc). If we dont want to give that control we can get rid of this argument everywhere....

Reply:
makes sense, our plan is to do something similar on the other SDKs as well